### ISSUE TITLE
RStudio Addin idea

### ISSUE THREAD

**Issue by sdhutchins** [ropenscilabs/syn#26]
I came across this package (and I love the idea)!

I was wondering if there was any thought about creating an addin for this and perhaps an interactive one? 

- [ ] Option 1: You can select the word you're looking for a synonym for and get suggestions. 

- [ ] Option 2:  The addin scans the entire document to get synonym suggestions at a specified readability level and returns synonym suggestions for words at that level.

**Comment by njtierney** [comment]
Hello!

I think that is a great idea - the only thing stopping me from jumping on this is that I think I would prefer the syn package to be dependency free. 

That said, I'm not really sure what the added benefit really is of having a dependency free package like this. I think that having addins would make the package more accessible and useful.

Someone else has actually implemented an add-in with `syn` - called `synamin` or something like that, I actually can't find it, unfortunately, I can't remember the name.

So, in short, both of your ideas are awesome, and I think they would be great to add - how would you feel about making a pull request to include these features? I would be happy to review your code. Then, if we decide that `syn` shouldn't include these features, you can create your own awesome package with these features?

Sorry that is a bit of a complex answer, but happy to discuss further :)

**Comment by sdhutchins** [comment]
Thanks for responding, @njtierney!

I located the package you mentioned, [synamyn](https://github.com/gadenbuie/synamyn).

It actually does a lot of what I hoped the addin would do outside of readability and searching the entire document to provide "suggestions".  I can certainly see the value of keeping a package like this dependency free, but I'll certainly look into making a PR or just chatting with the creator of synamyn (maybe both).

**Comment by njtierney** [comment]
OK, thanks @sdhutchins !
